docker/Dockerfile

```diff
@@ -118,5 +118,5 @@ RUN \
     build-locale-archive
 
 WORKDIR /
-ENTRYPOINT ["/usr/bin/contextBroker","-fg"]
-EXPOSE 1026
\ No newline at end of file
+ENTRYPOINT ["/usr/bin/contextBroker","-fg", "-multiservice"]
```

Comment:
It wasn't enabled by default in the RPM configuration, but that changed recenctly (don't remember if in version 0.23.0 or 0.22.0). It think it was requested by @dmoranj , maybe he can provide additional information about the rationale of the change.